### PR TITLE
Update deu.ini

### PR DESCRIPTION
**Pull request by MorpheusTP**
Translated! Also i fixed some spelling errors from SalmonDE.

- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
{Type something here}

## Optional information
{Type something here}

### CODE REVIEW

File: resources/deu.ini [hunk 1]
```diff
@@ -1,213 +1,207 @@
 # German language pack for MyPlot
-# Created by SalmonDE
+# Created by SalmonDE and morpheus132
```
Comment (jasonwynn10) [inline resources/deu.ini]: why not use your github name?

Comment (MorpheusTP) [inline resources/deu.ini]: Because I just use this name here. But OK lets use morpheusTP.

Comment (MorpheusTP) [inline resources/deu.ini]: *just only

Comment (jasonwynn10) [inline resources/deu.ini]: I'm just saying this if you would like credit if the project were ever removed from github

**Comment by MorpheusTP** [comment]
Do you update deu.ini now?

File: resources/deu.ini (hunk 1 above)
Comment (MorpheusTP) [inline resources/deu.ini]: Oh yeah you are right.  Please change it to morpheusTP

**Pull request merged**